Exam/Exam2.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Exam 2
+ * Cree un programa que dado un arreglo de números encuentre el primer elemento que se repite o -1 en caso de no haber elementos repetidos.
+
+1. Dado `[2, 1, 3, 5, 3, 2]` el primer elemento que se repite es `3`
+2. Dado `[2, 4, 3, 5, 1]` ya que no existe elementos repetidos el resultado es `-1`
+3. Dado `[2, 1, 1, 5, 3, 2]` el primer elemento que se repite es `1`
+4. Dado `[8, 4, 6, 2, 6, 4, 7, 9, 5, 8]` el primer elemento que se repite es `6`
+ */
+
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.io.Console;
+
+
+public class Exam2 {
+    public static void main(String[] args) {
+
+        Console console = System.console();
+
+        int NUMBER_OF_ELEMENTS = Integer.parseInt(console.readLine("How many elements do you want to read?: "));
```

Comment:
Awesome work, just remember that the notation with capital letter are reserved for constants, it make sense for NUMBER_OF_ELEMENTS to be constant but you need to add the final ad the beginning  `final int NUMBER_OF_ELEMENTS`